I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/main_room.rpy", line 61, in script call
    call auditorium from _call_auditorium
  File "game/auditorium.rpy", line 25, in script
    show charging_station zoom 0.7 at right
Exception: Image 'charging_station' does not accept attributes 'zoom 0 .7'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/main_room.rpy", line 61, in script call
    call auditorium from _call_auditorium
  File "game/auditorium.rpy", line 25, in script
    show charging_station zoom 0.7 at right
  File "D:\renpy-8.0.3-sdk.7z\renpy-8.0.3-sdk\renpy\ast.py", line 1408, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "D:\renpy-8.0.3-sdk.7z\renpy-8.0.3-sdk\renpy\ast.py", line 1371, in show_imspec
    renpy.config.show(name,
  File "D:\renpy-8.0.3-sdk.7z\renpy-8.0.3-sdk\renpy\exports.py", line 686, in show
    if not base.find_target() and renpy.config.missing_show:
  File "D:\renpy-8.0.3-sdk.7z\renpy-8.0.3-sdk\renpy\display\image.py", line 414, in find_target
    self.target = target._duplicate(a)
  File "D:\renpy-8.0.3-sdk.7z\renpy-8.0.3-sdk\renpy\display\core.py", line 397, in _duplicate
    args.extraneous()
  File "D:\renpy-8.0.3-sdk.7z\renpy-8.0.3-sdk\renpy\display\core.py", line 265, in extraneous
    raise Exception("Image '{}' does not accept attributes '{}'.".format(
Exception: Image 'charging_station' does not accept attributes 'zoom 0 .7'.

Windows-10-10.0.19045 AMD64
Ren'Py 8.0.3.22090809
BABEL 1.0
Tue Apr  4 11:55:17 2023
